app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    target_file, new_file = command.split()[1:]
```

Comment:
"Make sure you check that the cp input command is correct." it is from the **checklist**